packages/headless/src/controllers/core/facets/range-facet/core-range-facet-utils.ts

```diff
@@ -0,0 +1,14 @@
+interface AssertRangeFacetOptions {
```

Comment:
Boy scouting: this util function was in the headless-core-numeric-facet.ts file (which seemed wrong to me).